samples/excel/range/range.yaml

```diff
@@ -6,191 +6,236 @@ name: Range basics
-            range.format.autofitColumns();
+                let range = sheet.getRange("E3");
+                range.formulas = [[ "=C3 * D3" ]];
+                range.format.autofitColumns();
 
-            await context.sync();
-        });
+                await context.sync();
+            });
 
-        console.log("Done!");
+            console.log("Done!");
+        }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
     }
 
     async function setFormulas() {
-        await Excel.run(async (context) => {
-            let sheet = context.workbook.worksheets.getActiveWorksheet();
-
-            let data = [
-                ["Total Price"],
-                ["=C3 * D3"],
-                ["=C4 * D4"],
-                ["=C5 * D5"],
-                ["=SUM(E3:E5)"]
-            ];
-            
-            let range = sheet.getRange("E2:E6");
-            range.formulas = data;
-            range.format.autofitColumns();
-
-            await context.sync();
-        });
-
-        console.log("Done!");
+        try {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getActiveWorksheet();
+
+                let data = [
+                    ["Total Price"],
+                    ["=C3 * D3"],
+                    ["=C4 * D4"],
+                    ["=C5 * D5"],
+                    ["=SUM(E3:E5)"]
+                ];
+                
+                let range = sheet.getRange("E2:E6");
+                range.formulas = data;
+                range.format.autofitColumns();
+
+                await context.sync();
+            });
+
+            console.log("Done!");
+        }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
     }
 
     async function setFormulasR1C1() {
-        await Excel.run(async (context) => {
-            let sheet = context.workbook.worksheets.getActiveWorksheet();
-
-            let data = [
-                ["Total Price"],
-                ["=R[0]C[-2] * R[0]C[-1]"],
-                ["=R[0]C[-2] * R[0]C[-1]"],
-                ["=R[0]C[-2] * R[0]C[-1]"],
-                ["=SUM(R[-3]C[0]:R[-1]C[0])"]
-            ];
-            
-            let range = sheet.getRange("E2:E6");
-            range.formulasR1C1 = data;
-            range.format.autofitColumns();
-
-            await context.sync();
-        });
-
-        console.log("Done!");
+        try {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getActiveWorksheet();
+
+                let data = [
+                    ["Total Price"],
+                    ["=R[0]C[-2] * R[0]C[-1]"],
+                    ["=R[0]C[-2] * R[0]C[-1]"],
+                    ["=R[0]C[-2] * R[0]C[-1]"],
+                    ["=SUM(R[-3]C[0]:R[-1]C[0])"]
+                ];
+                
+                let range = sheet.getRange("E2:E6");
```

Comment:
Yes, but it makes the snippet a bit more complicated. Per previous comment I would like a less steep ramp.